src/corva/handlers.py

```diff
@@ -52,6 +57,17 @@ def wrapper(aws_event: Any, aws_context: Any) -> List[Any]:
             user_handler=handler,
             logger=CORVA_LOGGER,
         ) as logging_ctx:
+            raw_custom_event_type: Optional[
+                Type[RawPartialRerunMergeEvent]
+            ] = _get_custom_event_type_by_raw_aws_event(aws_event)
+            custom_handler = HANDLERS.get(raw_custom_event_type)
+            if raw_custom_event_type and custom_handler is None:
```

Comment:
Can we make sdk log warning in case if "partial rerun merge event comes and there is no handler defined".

With current implementation warning will never be issued.

Reply:
Thanks for spotting this out. I've changed the implementation.